webpack.config.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line import/extensions
+const config = require('10up-toolkit/config/webpack.config.js');
+
+const DependencyExtractionWebpackPlugin = require('@wordpress/dependency-extraction-webpack-plugin');
+
+module.exports = {
+	...config,
+	plugins: [
+		...config.plugins,
+		new DependencyExtractionWebpackPlugin({
+			requestToExternal: () => undefined,
+			requestToHandle: () => undefined,
+		}),
+	],
+};
```

Comment:
@nicholasio Do you have any idea what I might be doing wrong here? this is not getting picked up 🤔